src/charts/mini-tooltip.js

```diff
@@ -378,6 +378,22 @@ define(function(require){
             return this;
         }
 
+        /**
+         * Gets or Sets the formatter function for the value displayed on the tooltip.
+         * Setting this property makes the tooltip ignore numberFormat. Set by default to
+         * d3-format formatter with numberFormat.
+         * @param  {Function} _x Desired formatter function
+         * @return {Function | module} Current valueFormatter or Chart module to chain calls
+         * @public
+         */
+        exports.valueFormatter = function(_x) {
+            if (!arguments.length) {
+                return valueFormatter;
+            }
+            valueFormatter = _x;
+            return this;
```

Comment:
Nit: space before return

Reply:
You mean to add an empty line before? I copied the structure of other getter/setter methods in the source and I haven't seen an empty line there. If you want me to add it please confirm.

Reply:
Hey @adrm - the formatting in this file is wrong. Please do add an empty line before return. If you want you can also change the format in the same file to keep it consistent. Thanks a lot!

Reply:
Thanks for the explanation. Will do!

Reply:
fixed